README.md

```diff
@@ -14,36 +14,133 @@
 
 ----
 
-OWASP [CycloneDX][link_website] is a full-stack Bill of Materials (BOM) standard
-that provides advanced supply chain capabilities for cyber risk reduction.
+OWASP [CycloneDX][link_website] is a full-stack Bill of Materials (BOM) standard that provides advanced supply chain capabilities for cyber risk reduction.
 
-This Python package provides data models, validators and more, 
-to help you create/render/read CycloneDX documents.
+This Python package provides data models and tools for working with CycloneDX documents.
 
-**This package is not designed for standalone use. It is a software library.**
+> **Note**: This package is a software library not intended for standalone use. For generating Software Bill of Materials (SBOM), check out [CycloneDX Python][cyclonedx-python] or [Jake][jake].
 
-As of version `3.0.0`, the internal data model was adjusted to allow CycloneDX VEX documents to be produced as per
-[official examples](https://cyclonedx.org/capabilities/bomlink/#linking-external-vex-to-bom-inventory) linking VEX to a separate CycloneDX document.
+## Installation
 
-If you're looking for a CycloneDX tool to run to generate (SBOM) software bill-of-materials documents, why not checkout 
-[CycloneDX Python][cyclonedx-python] or [Jake][jake].
+**Via pip:**
+```shell
+pip install cyclonedx-python-lib
+```
 
-## Documentation
-
-View the documentation [here](https://cyclonedx-python-library.readthedocs.io/).
+**Via Conda:**
+```shell
+conda install -c conda-forge cyclonedx-python-lib
+```
 
 ## Python Support
 
-We endeavour to support all functionality for all [current actively supported Python versions](https://www.python.org/downloads/).
+We endeavor to support all functionality for all [current actively supported Python versions](https://www.python.org/downloads/).
 However, some features may not be possible/present in older Python versions due to their lack of support.
 
-## Changelog
+## VEX Support
+
+As of version `3.0.0`, the library supports CycloneDX VEX documents production with [official example](https://cyclonedx.org/capabilities/bomlink/#linking-external-vex-to-bom-inventory) compatibility for linking VEX to separate CycloneDX documents.
+
+## Documentation
 
-See our [CHANGELOG][chaneglog_file].
+Complete documentation is available on [Read the Docs][link_rtfd]. This includes:
+- API Reference
+- Usage Examples
+- Integration Guides
+- Best Practices
+
+## Responsibilities
+
+* Provide a general-purpose Python implementation of [CycloneDX][link_website]
+* Provide type hints for implementation support
+* Support JSON/XML document parsing and generation
+* Validate CycloneDX documents against schema specifications
+* Support multiple CycloneDX specification versions
+* Maintain comprehensive data models for BOM manipulation
+* Enable pip-based installation for downstream usage
+
+## Capabilities
+
+### Enums
```

Comment:
the description next to the enum types do not add much value, as they duplicate the enums name and add the word `type`. 
i'd rather remove the additional descriptions.